src/main/java/RacingApplication.java

```diff
@@ -0,0 +1,12 @@
+import controller.RacingController;
+import domain.RandomNumberGenerator;
+
+public class RacingApplication {
+
+    public static void main(String[] args) {
+
+        new RacingController().run(new RandomNumberGenerator());
+
+    }
```

Comment:
줄바꿈도 의도가 담길 수 있어요. 필요 없는 줄바꿈이라면, 오해하지 않게 깔끔하게 관리해주세요
```suggestion
    public static void main(String[] args) {\
        new RacingController().run(new RandomNumberGenerator());
    }
```